### PR TITLE
Clean up & New pages

### PR DESCRIPTION
**Pull request by AsherSpurr**
# Pull Request
 Tea education pages, clean up of styling, fetch function for single tea
## Description
Adds two new components, TeaEd and TeaArticle. Adds fetch function for a single tea. Formats document and removed unused comments
## Related Issues
Updates tea blends button to be blends. 
## Screenshots (if applicable)
Screenshot 2024-04-06 at 6 41 50 PM
Screenshot 2024-04-06 at 6 41 06 PM

## Changes Made
**Additions:**
- TeaEd component - contains logic for its child TeaArticle
- Adds new interface for hardcoded Tea info/education
- dynamic routing to and from TeaArticle pages
- fetch function for single tea -- not integrated/unused

**Removals:**
- empty space
- unused comments
## Testing
No testing added for these pages yet
## Checklist
- [x] The code follows the project's coding standards.
- [ ] Unit tests have been added or updated to cover the changes.
- [x] Documentation has been updated to reflect the changes (if applicable).
- [x] The code compiles without errors.
- [ ] The changes have been tested locally and pass all relevant tests.
- [ ] All new and existing tests pass.
- [x] The pull request has been reviewed by at least one other contributor.
## Additional Information
**New components are completely unstyled**

### CODE REVIEW

**Review by corysanders3 — APPROVED** [review]
Love all the tea hardcoded info!! Awesome detail. Really appreciate that.

File: src/components/home/Home.tsx [hunk 1]
```diff
@@ -1,54 +1,53 @@
...
-}, [navigate, setIsRedirected])
+    setIsRedirected(false);
+  }, [navigate, setIsRedirected]);
 
   return (
     <div>
       <div className="intro-wrapper">
         <article className="article-intro">
-          <img src={teas} />
+          <img src={teas} alt='Cupcake baking pan with different tea blends filling each cup' />
```
Comment (tednaphil) [inline src/components/home/Home.tsx:26]: Yay for descriptive alt text!!

File: src/components/home/Home.tsx [hunk 2]
```diff
@@ -1,54 +1,53 @@
...
-            <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
-              ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-              aliquip ex ea commodo consequat.
+            <h2 className="home-h2">Let's 'spill the tea' on tea!</h2>
+            <p className="info-desc">
+              Did you know that all 'teas' originate from the leaves of the Camellia
+              Sinensis plant?(break this up) Any 'tea' that is not from this plant is actually an herbal blend! (break here) What makes each tea unique is
+              the creation process is goes through, including:
```
Comment (tednaphil) [inline src/components/home/Home.tsx:32]: This is fabulously engaging content! Thank you for working on getting the intros together @AsherSpurr !

File: src/utils/interface.tsx [hunk 3]
```diff
@@ -19,4 +19,111 @@ export interface Tea {
...
+    origin: string,
+    creation: string,
+    withering?: string,
+    rolling?: string,
+    oxidation: string,
+    heating: string,
+    climate: string
+}
+
+export const allHardcodedTeas: TeaHardcoded[] = [
```
Comment (tednaphil) [inline src/utils/interface.tsx:84]: WHOA! 💯

**Review by tednaphil — CHANGES_REQUESTED** [review]
This all looks good but it seems like there are some merge conflicts to be resolved before it's able to be merged to main. If yall want to sift through the conflicts in the vscode merge editor by merging main into this branch beforehand, we can start with that Sunday morning!

**Review by tednaphil — APPROVED** [review]

**Pull request merged**